manifests/stub.pp

```diff
@@ -17,7 +17,7 @@
     content => template('unbound/stub.erb'),
   }
 
-  if $insecure == true {
+  if str2bool($insecure) == true {
```

Comment:
> str2bool: This converts a string to a boolean. This attempts to convert strings that contain values such as '1', 't', 'y', and 'yes' to 'true' and strings that contain values such as '0', 'f', 'n', and 'no' to 'false'. Type: rvalue

It will also leave things that are already booleans intact. I have tested this on the strings "true" and "false" and it works fine.